README.md

```diff
@@ -602,10 +602,12 @@ we can use stack defined as follows:
 stacks:
   us-east-1:
     my-stack:
-      sparkle_pack_template: template_name
+      template: template_name
```

Comment:
Just wanting to confirm my understanding is correct and that `template` includes the template's file extension?

Reply:
It usually does, but for sparkle pack templates it refers to the symbol key used in `SparkleFormation.new(:template_name) do`

Reply:
That's unfortunate 😞